contracts/PoolLens.sol

```diff
@@ -597,4 +560,32 @@ contract PoolLens is Initializable {
       else return (collateralValue * 1e18) / (collateralValue - liquidity);
     }
   }
+
+  function getHealthFactorHypothetical(
+    IonicComptroller pool,
+    address account,
+    address cTokenModify,
+    uint256 redeemTokens,
+    uint256 borrowAmount,
+    uint256 repayAmount
+  ) external view returns (uint256) {
+    (uint256 err, uint256 collateralValue, uint256 liquidity, uint256 shortfall) = pool.getHypotheticalAccountLiquidity(
+      account,
+      cTokenModify,
+      redeemTokens,
+      borrowAmount,
+      repayAmount
+    );
+
+    if (err != 0) revert ComptrollerError(err);
```

Comment:
duplicate code, you can use `getHypotheticalAccountLiquidity` with 0 for `borrowAmount` and `repayAmount` instead